libpretixsync/src/main/java/eu/pretix/libpretixsync/api/PretixApi.java

```diff
@@ -208,6 +244,9 @@ private ApiResponse apiCall(Request request, boolean json) throws ApiException,
             throw new ApiException("Server error: Resource not found.");
         } else if (response.code() == 304) {
             throw new ResourceNotModified();
+        } else if (response.code() == 400) {
+            response.close();
+            throw new ApiException("This initialization token has already been used.");
```

Comment:
I'm not sure if it makes sense to use the same postResource() here, as this message only makes sense for the initialization request, while an error 400 can be returned for a multitude of ohter reasons.

Reply:
you are right, will introduce separate method for that